tasks/main.yml

```diff
@@ -1,7 +1,7 @@
 ---
 - name: Add PowerDNS APT key
   apt_key:
-    url: https://repo.powerdns.com/FD380FBB-pub.asc
+    url: https://repo.powerdns.com/CBC8B383-pub.asc
```

Comment:
According to the pdns website, the repo is still signed by a key with the former id: https://repo.powerdns.com/
We just need to redownload it.

Reply:
After digging further into the topic you might be right, e.g. ubuntu key server lists both keys (http://keyserver.ubuntu.com/pks/lookup?op=vindex&search=0x1b0c6205fd380fbb) so apparently you actually can use the same key and refresh it or something like that